sci-mathematics/pymc/pymc-2.3.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Markov Chain Monte Carlo sampling toolkit"
+HOMEPAGE="https://github.com/${PN}-devs/${PN} https://pypi.python.org/pypi/${PN}"
+SRC_URI="https://github.com/${PN}-devs/${PN}/tarball/v${PV} -> ${P}.tar.gz"
+
+SLOT=0
+KEYWORDS="~amd64-linux ~x86-linux"
```

Comment:
...you're only using Gentoo from prefix? If so, I can keyword them for `x86` and `amd64` later.